src/traces/sankey/render.js

```diff
@@ -67,6 +67,30 @@ function sankeyModel(layout, d, traceIndex) {
         Lib.warn('node.pad was reduced to ', sankey.nodePadding(), ' to fit within the figure.');
     }
 
+    // Create transient nodes for animations
+    Object.keys(calcData._groupLookup).forEach(function(nodePointNumber) {
```

Comment:
`for(var k in calcData._groupLookup)` is generally faster.

Reply:
Done in 1cd206f7fdba75b010650331fc1f7a726ec488df  :racehorse: